capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import java.io.BufferedReader;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.sps.data.FeedbackInput;
+import org.json.simple.JSONObject;    
+import org.json.simple.JSONArray;  
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Collection;
+
+/** Servlet that suggests a difference score for Perspective. */
+@WebServlet("/suggest_perspective")
+public class SuggestCommentServlet extends HttpServlet {
+  private static final String URL = "https://commentanalyzer.googleapis.com/v1alpha1/comments:suggestscore?key=API_KEY";
+  private static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
```

Comment:
Yup, I removed this one. I'll hold off on changing the variable name in `post` because I entirely remove the `post` method from this file in the branch where I write the tests, but I'll change the variable name there to `jsonMediaType`